spec/simplecov_setup.rb

```diff
@@ -5,7 +5,7 @@
 
   SimpleCov.root(File.expand_path('../..', __FILE__))
   SimpleCov.start do
-    minimum_coverage 100
+    minimum_coverage 99.71
```

Comment:
😱

Reply:
😊

Reply:
It's only [this one line](https://github.com/ad2games/soapy_bing/pull/12/files#diff-8c841fa73c1f26de8e5e83a5c72cca65R37).

Reply:
How about a :no:cow:: instead?

Reply:
:roll_eyes: